cmd/find.go

```diff
@@ -79,13 +125,26 @@ var clusterCmd = &cobra.Command{
 		if err != nil {
 			klog.Fatalf("Error getting helm releases from cluster: %v", err)
 		}
+		outputFile := viper.GetString("output-file")
 		if outputFile != "" {
 			err = out.ToFile(outputFile)
 			if err != nil {
 				panic(err)
 			}
 		} else {
-			out.Print(wide)
+			out.Print(viper.GetBool("wide"))
+		}
+	},
+}
+
+var genConfigCmd = &cobra.Command{
```

Comment:
seems like this should probably get its own cmd file

Reply:
It seemed a little odd to do that since all of the existing configuration options are already in that file. 

In fact, I would almost vote to move all of the find.go into root.go and remove find.go